flake.nix

```diff
@@ -670,6 +670,12 @@
         pkgs.mkShell {
           buildInputs = [ terraform run-terraform update-infra ];
         };
+
+      stdlibTests = pkgs.runCommandLocal "stdlib-test" { }
+        ''
+          ${pkgs.lib.getExe self.packages."${system}".default} test ${./core/stdlib/std.ncl};
```

Comment:
Isn't the `;` at the end going to make the command pass whatever is the actual result? Maybe a sanity check would be to push a commit with a deliberate error in the stdlib somewhere, just to be sure.

Reply:
I don't know exactly why (maybe `runCommandLocal` does a `set -xeu` or something?) but it seems to fail if I make it fail.

Reply:
But why do you need the `;` ? Can't we use a `&&`? Do we also have to necessarily create `$out` ?

Reply:
Sure, I can use `&&` instead; the behavior seems to be the same.

Yes, without making `$out` then `nix flake check` fails with
```
error: builder for '/nix/store/kyr0xs7ncr05i3frnphvvvr1q8p8ja1c-stdlib-test.drv' failed to produce output path for output 'out' at '/nix/store/kyr0xs7ncr05i3frnphvvvr1q8p8ja1c-stdlib-test.drv.chroot/root/nix/store/yil1zc0j0vy5p3sjsp0c1kqi8r0h6krj-stdlib-test'
```

Reply:
Thanks, looks good! I think `&&` is indeed better as it doesn't rely on assumptions about how `runCommandLocal` works internally :+1: